migrations/versions/6ddd7dbd2138_.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: 6ddd7dbd2138
+Revises: 8bea1f7d2e44
+Create Date: 2021-05-24 15:55:07.285110
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '6ddd7dbd2138'
+down_revision = '8bea1f7d2e44'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+import app
+import app.extensions
+
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('job_control',
+    sa.Column('created', sa.DateTime(), nullable=False),
+    sa.Column('updated', sa.DateTime(), nullable=False),
+    sa.Column('viewed', sa.DateTime(), nullable=False),
```

Comment:
Considering whether we should have a 'completed' DateTime as well to measure turnaround time. Sage does keep this information for the job however.

Reply:
In which case I'd argue that we shouldn't duplicate the info unless there is a definite need

Reply:
i agree here -- in part because this information is not something the user necessarily needs (quickly or often) so the cost of asking sage is not a blocker.   i wonder though:  does sage _really_ keep this forever, or is there a chance the queue gets flushed of completed jobs?  i vaguely recall this happening now and then.  maybe a question for @bluemellophone .